selfdrive/car/ford/fordcan.py

```diff
@@ -128,17 +128,17 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
   Frequency is 50Hz.
   """
 
-  decel = accel < 0 and long_active
+  actuateBrakes = gas == -5 and long_active
```

Comment:
I don't think we verified that we could get very much engine braking which is why we checked if accel was less than 0 m/s^2 here. This now relies solely on engine braking from 0 to -0.5 m/s^2 which is fairly high. We should check if the system can match something like -0.45 m/s^2, we'd never actuate brakes until the controller winds up now.

Here is an example where this caused a huge braking deadzone for GM which we fixed by lowering the engine braking threshold: https://github.com/commaai/openpilot/pull/25354

Reply:
We'll probably be able to tell better tomorrow after getting the routes. But the EVs even flip off 1 pedal drive when using BlueCruise so it's basically driving as an ICE vehicle would, so I think the look ups should be similar/same against ICE/EVs, at least.

Reply:
It would make sense if the PCM/ECM uses regen on EVs, but we should check (since GM's Bolt does not, and relying on it would also introduce some deadzone)